stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2522,12 +2522,12 @@ final internal class _Native${Self}StorageImpl<${TypeParameters}> :
   }
 
   @_versioned
-  internal var _capacity: Int {
+  internal var capacity: Int {
```

Comment:
I recommend keeping these underscored (everything internal is by convention underscored.)